src/lib/setup/pipelineService.ts

```diff
@@ -149,6 +162,22 @@ export const pollForPipelineStatus = async (
   } while (!build || build.result === 0);
 };
 
+const deletePipelineIfExist = async (
+  buildApi: IBuildApi,
+  rc: RequestContext,
+  pipelineName: string
+): Promise<void> => {
+  const pipeline = await getPipelineByName(
+    buildApi,
+    rc.projectName,
+    pipelineName
+  );
+  if (pipeline && pipeline.id) {
+    logger.info(`${pipelineName} is found, deleting it`);
```

Comment:
Should make explicit that the pipeline named `${pipelineName}` was found:

logger.info(`Pipeline ${pipelineName} was found - deleting pipeline`);

Reply:
accept